### ISSUE TITLE
The Importance of Rchain Meetups and guidelines to organizing one

### ISSUE THREAD

**Issue by Preshcode** [rchain/bounties#772]
Benefit to RChain

Networking in the digital age is very important but always being reminded that in person networking is still a key to success. I created an article that explains why Rchain Meetups are essential for the growth of the platform and also gave guidelines on how to organize a successful Rchain meetup. 

Budget and Objective

Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely.

Estimated Budget of Task: $[470]
Estimated Timeline Required to Complete the Task: [completed]

Legal

Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.

**Comment by Preshcode** [comment]
Link to the article <a href="http://www.nairaland.com/4452510/rchain-community#68440266>link</a>

**Comment by Preshcode** [comment]
Link to the article <a href="http://www.nairaland.com/4452510/rchain-community#68440266">link</a>

**Comment by nonnykul** [comment]
That's a good write up

**Comment by jasoncruzzy** [comment]
Nice one comprehensive, i will post it on Reddit(RChain_Africa)

**Comment by dckc** [comment]
How is this different from https://github.com/rchain/bounties/wiki/local-meetup-program:-summary ?

There should at least be links in both directions.

**Comment by nonnykul** [comment]
The aim is to promote the writeup on different platforms.

**Comment by Preshcode** [comment]
They are quite different @dckc. While the local Meetup program summary only listed the materials, prerequisites, and budget for Meetup, this article I wrote went ahead to discuss the essence of Rchain Meetups in the growth of Rchain tech globally, Precise guidelines to Organizing the Meetups and finally examples of Blockchain technology meetup organizers and their experiences.
But since both contains info about Rchain Meetup.. What I'll do is to include the link to the local Meetup program summary in the article.

**Comment by jimscarver** [comment]
It looks like a good writeup. I suggest it should include relevant links to rchain info and the bounty system.

It  It seems relevant to members generally so promoting just to africa seems to limit the value generated from the effort. 

This is a good start however and I will support the budget for $350 for now.

**Comment by zsluedem** [comment]
I think this one is kind of related to #779 . @Preshcode Would you like to work with #779 ?We are trying to update https://github.com/rchain/bounties/wiki/local-meetup-program:-summary

**Comment by Preshcode** [comment]
Great point @jimscarver. I'll work towards promoting the article in other Rchain media channels.

**Comment by Preshcode** [comment]
Yeah @zsluedem. Am interested in working on the update with ur team. 
Need your assistance with voting for this issue though.
